code/modules/mob/living/simple_animal/hostile/space_dragon.dm

```diff
@@ -696,7 +698,7 @@
 			return FALSE
 		is_listed = TRUE
 	var/carp_ask = tgui_alert(usr,"Become a carp?", "Help bring forth the horde?", list("Yes", "No"))
```

Comment:
```suggestion
	var/carp_ask = tgui_alert(user, "Become a carp?", "Carp Rift", list("Yes", "No"))
```
L + message / title ratio + spacing + user

Reply:
dont touch the message its funny even if it isnt as true anymore

Reply:
Id rather they be swapped than the title longer than the message

Reply:
The message is the second argument, for context